spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+require './lib/vehicle'
+require './lib/vehicle_factory'
+require './lib/dmv_data_service'
+require './lib/facility_factory'
+
+RSpec.describe FacilityFactory do
+
+    let(:oregon) do
```

Comment:
This is actually a GREAT practice of using test data in this test rather than calling the DmvDataFactory. Good call! This way you know exactly what data is being passed into your tests.

Reply:
In mod3, when you are handling lots of data like this, we will show you some other ways to store data in a test but for now this is totally fine. I wouldn't say this is too long of a test file or there is too much going on